src/main/java/io/vertx/kafka/admin/impl/KafkaAdminClientImpl.java

```diff
@@ -564,4 +565,46 @@ private void describeLogDirsInternal(List<Integer> brokerIds, Promise<Map<Intege
       promise.fail(e);
     }
   }
```

Comment:
Changes in describeLogDirsInternal method were made due to describeLogDirsResult.all() method being deprecated after changing kafka library version